src/west/app/project.py

```diff
@@ -428,6 +428,9 @@ def do_add_parser(self, parser_adder):
             epilog=f'''\
 {ACTIVE_PROJECTS_HELP}
 
+Note: To list only inactive projects you can use --inactive. Giving an explicit
```

Comment:
The only question in my mind about this feature is whether `--inactive` should filter out projects when they are explicitly provided by the user. No particular preference from my side.

So, should:
`west list -i my-foo-active my-bar-inactive` print both foo and bar or only bar?

Reply:
My first thought would be to have only print bar, so `-i` would invert the printed projects.

Reply:
But remember that `west list foo bar` prints **both** `foo` and `bar`!

Reply:
Yes, I think it's the "safe" choice here.

Reply:
Make sense to disallow that combination, asking which of list of projects is inactive does not sound like a very useful thing to do to me.

Reply:
Implemented.